app/styles/ice-pop/index.scss

```diff
@@ -4,9 +4,6 @@
 //
 // Styleguide Ice Pop
 
-// Imported Ice Box Styles
```

Comment:
I moved this to dummy app css (bc it actually needs ice-box for UI stuff), otherwise this would force a project using the ice-pop addon to include all of ice-box, which we don't need to worry about anymore due to the updates I made.